### DISCUSSION TITLE
[@types/tabulator-tables] .selectRow() should also accept single RowLookup (not only arrays of RowLookup)

### DISCUSSION THREAD

**Discussion by toneb** [DefinitelyTyped/DefinitelyTyped#68697] [Issues with a @types package] [answered]
Current signature:
selectRow: (lookup?: RowLookup[] | RowRangeLookup | true) => void;

Should be:
selectRow: (lookup?: RowLookup[] | **RowLookup** | RowRangeLookup | true) => void;

Documentation:
https://tabulator.info/docs/5.6/select#manage

[a comment by an automated account is not shown]

**Comment by mike-lischke** [comment]
@toneb, can you please file a pull request to change this?

**Reply by toneb** [reply, marked as answer]
I created PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68701. I also improved the deselectRow and RowLookup definitions.

**Reply by mike-lischke** [reply]
Perfect, thanks.
